lib/git_commit_notifier/emailer.rb

```diff
@@ -221,6 +221,13 @@ def send
     date = @offset_date     # Date notifier started, plus 1 second per commit processed
     date = Time.new.rfc2822 if date.nil?    # Fallback to current date if date is nil
 
+    # Check if the recepients are set to be sent via Bcc and set the To to the commit
+    # author
+    if config['bcc_recepients'] && config['delivery_method'] != 'nntp'
```

Comment:
I didn't want to change your To logic above, it's quite clean and this is a simple override if Bcc is set.